util/src/main/java/io/kubernetes/client/util/generic/GenericKubernetesApi.java

```diff
@@ -1159,6 +1159,7 @@ private CallBuilder makeNamespacedDeleteCallBuilder(
             namespace,
             this.resourcePlural,
             name)
+                .body(deleteOptions)
                 // TODO: fill/convert the option
```

Comment:
Should we remove this TODO as this is no more valid?

Reply:
Yes, please delete this now.